code-challenges.test.js

```diff
@@ -1,51 +1,160 @@
-// ASSESSMENT 2: Coding Practical Questions with Jest
+// // ASSESSMENT 2: Coding Practical Questions with Jest
 
-// Please read all questions thoroughly
-// If you get stuck, leave comments to help us understand your thought process
+// // Please read all questions thoroughly
+// // If you get stuck, leave comments to help us understand your thought process
 
-// Use test driven development to complete the following questions
-// Add appropriate dependencies: $ yarn add jest
+// // Use test driven development to complete the following questions
+// // Add appropriate dependencies: $ yarn add jest
 
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
+// // Reminder: The test will call your function
+// // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
+// // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
 
-// b) Create the function that makes the test pass.
+// describe ('multiplyByThree', () => {
+//     it('that takes in an array of numbers and returns an array with all the numbers multiplied by 3', () => {
+
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
+
+// expect(multiplyByThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+// expect(multiplyByThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+// })
+// })
```

Comment:
Pay attention to your indentation. This code is very hard to read, which can be easily fixed with proper indentation!